assets/js/modules/analytics-4/components/audience-segmentation/dashboard/NoAudienceBannerWidget/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * NoAudienceBannerWidget component.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import whenActive from '../../../../../../util/when-active';
+
+/**
+ * Internal dependencies
+ */
+import { useSelect } from 'googlesitekit-data';
+import { MODULES_ANALYTICS_4 } from '../../../../datastore/constants';
+import NoAudienceBanner from './NoAudienceBanner';
+
+function NoAudienceBannerWidget( { Widget, WidgetNull } ) {
+	const availableAudiences = useSelect( ( select ) => {
+		const audiences = select( MODULES_ANALYTICS_4 ).getAvailableAudiences();
+		return audiences?.map( ( audience ) => audience.name );
+	} );
+	const configuredAudiences = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS_4 ).getConfiguredAudiences()
+	);
```

Comment:
I think both of these should use the `useInViewSelect` hook as they are in the widget area and their resolvers will make API requests. What do you think?